tests/vmi_multus_test.go

```diff
@@ -1122,10 +1138,9 @@ var _ = Describe("[Serial]Macvtap", func() {
 	})
 })
 
-func cidrToIP(cidr string) string {
+func cidrToIP(cidr string) (string, error) {
 	ip, _, err := net.ParseCIDR(cidr)
-	Expect(err).ToNot(HaveOccurred(), "Should be able to parse IP and prefix length from CIDR")
-	return ip.String()
+	return ip.String(), err
```

Comment:
Would you keep the error message here?
```go
fmt.Errorf("Failed to parse IP and prefix length from CIDR: %w", err)
```
Then we could drop the duplicated message from all the asserts and keep them as simple

```go
Expect(err).ToNot(HaveOccurred())
```

Reply:
I do not why, but I somehow understood error logging should be inside the test as well.
Now, this makes much more sense. Thanks, I will add a fix today.

Reply:
There are no hard rules. To me it looks like a good compromise to keep it short while asserting outside the helper.

Oh and edit on the message - it should be "failed to parse IP from CIDR" - the function does not care about the prefix length